### ISSUE TITLE
Error when trying to delete submissions from command line

### ISSUE THREAD

**Issue by idawgik** [verbb/formie#890]
**Description**
I'm setting up a new form on an in development site, and while building out the form (which is multi-page) I tested but didn't finish submitting a form. Now that I have everything the way I want with the field setup, I'm discovering an error related to submissions. As an aside, part of this was trying to do conditionals in a repeater, which when I discovered didn't work, I removed the conditional options on those fields.

When I try and view submissions in the admin, I get a system error.

```
error: "Invalid field UUID: c24767ce-dd15-4f29-92d1-6dfcfa2c4265"
exception: "yii\\base\\InvalidConfigException"
file: "/path/to/site/vendor/craftcms/cms/src/fields/conditions/FieldConditionRuleTrait.php"
line: 70
```

I consequently tried to delete submissions from the command line, but that isn't working. I first tried the `formie/gc/prune-incomplete-submissions` command, which didn't clear anything out.

I then tried the `formie/submissions/delete` which returns this error:

```
./craft formie/submissions/delete --form-handle onlineRfp
Exception 'Error' with message 'Typed property verbb\formie\console\controllers\SubmissionsController::$formHandle must not be accessed before initialization'

in /path/to/site/vendor/yiisoft/yii2/console/Controller.php:153

Stack trace:
#0 /path/to/site/vendor/yiisoft/yii2/base/Module.php(552): yii\console\Controller->runAction()
#1 /path/to/site/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction()
#2 /path/to/site/vendor/craftcms/cms/src/console/Application.php(90): yii\console\Application->runAction()
#3 /path/to/site/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction()
#4 /path/to/site/vendor/yiisoft/yii2/base/Application.php(384): yii\console\Application->handleRequest()
#5 /path/to/site/craft(13): yii\base\Application->run()
#6 {main}
```

I've tried with both form handle and form id on that command and same error (just changes from `$formHandle` to `$formId`).

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 2.0.0-beta.6
- Craft version: Craft 4.0.2
- Multi-site: Yes

**Comment by engram-design** [comment]
So I'll mention that the `formie/gc/prune-incomplete-submissions` command runs through Craft's garbage collection system, so it's not guaranteed that it'll be deleted straight away. I have however fixed the error you were getting on the command line in the latest `craft-4` branch. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.6",
  "...": "..."
}
```

Then run `composer update`. 

However, going back to your original issue - that's an interesting one, in that it references Craft' own native conditions handling `/path/to/site/vendor/craftcms/cms/src/fields/conditions/FieldConditionRuleTrait.php`, which shouldn't be the case, as we don't use it at all. I'll investigate that further!

**Comment by idawgik** [comment]
Thanks, I updated to that version and it did indeed fix the issue with the command line option. I was able to delete my 1 test submission.

Also, I just discovered what is causing the first error. In testing out new Craft 4 features, I had set up a custom source in Formie submissions. When viewing that list, I get that first error. Interestingly enough, when I go in to Customize to try and remove that, that error happens again, which prevents that whole panel from loading so I can't even delete it.

It seems somewhere along the line, I'd deleted a field, and it broke things. (I even just tried rebuilding the project config from command line, and that's failing about that same field).

So it looks like, at the moment, I managed to break something in Craft that's not necessarily your plugin. Thanks for looking into it all though.
